os/arch/xtensa/include/arch.h

```diff
@@ -0,0 +1,98 @@
+/******************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+/****************************************************************************
```

Comment:
Is this come from Nuttx? Back-ported? If it isn't, let's remove Nuttx lincense.

Reply:
Please check all of files which you pushed. If there are Nuttx codes, we must add Nuttx license as well. But like this file, there is no specific code which comes from Nuttx, we don't need to add it.